File: app/src/main/java/com/centaury/cataloguemovie/data/remote/detail/tvshow/SeasonsItem.java
package com.centaury.cataloguemovie.data.remote.detail.tvshow;

import com.google.gson.annotations.SerializedName;

import javax.annotation.Generated;

@Generated("com.robohorse.robopojogenerator")
public class SeasonsItem {

    @SerializedName("air_date")
    private String airDate;

    @SerializedName("overview")
    private String overview;

    @SerializedName("episode_count")
    private int episodeCount;

    @SerializedName("name")
    private String name;

    @SerializedName("season_number")
    private int seasonNumber;

    @SerializedName("id")
    private int id;

    @SerializedName("poster_path")
    private String posterPath;

    public String getAirDate() {
        return airDate;
    }

    public void setAirDate(String airDate) {
        this.airDate = airDate;
    }

    public String getOverview() {
        return overview;
    }

    public void setOverview(String overview) {
        this.overview = overview;
    }

    public int getEpisodeCount() {
        return episodeCount;
    }

    public void setEpisodeCount(int episodeCount) {
        this.episodeCount = episodeCount;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getSeasonNumber() {
        return seasonNumber;
    }

    public void setSeasonNumber(int seasonNumber) {
        this.seasonNumber = seasonNumber;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public void setPosterPath(String posterPath) {
        this.posterPath = posterPath;
    }

    @Override
    public String toString() {
        return
                "SeasonsItem{" +
                        "air_date = '" + airDate + '\'' +
                        ",overview = '" + overview + '\'' +
                        ",episode_count = '" + episodeCount + '\'' +
                        ",name = '" + name + '\'' +
                        ",season_number = '" + seasonNumber + '\'' +
                        ",id = '" + id + '\'' +
                        ",poster_path = '" + posterPath + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/centaury/cataloguemovie/data/remote/detail/tvshow/SeasonsItem.kt
package com.centaury.cataloguemovie.data.remote.detail.tvshow

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class SeasonsItem(

    @Json(name = "air_date")
    val airDate: String? = null,

    @Json(name = "overview")
    val overview: String? = null,

    @Json(name = "episode_count")
    val episodeCount: Int? = null,

    @Json(name = "name")
    val name: String? = null,

    @Json(name = "season_number")
    val seasonNumber: Int? = null,

    @Json(name = "id")
    val id: Int? = null,

    @Json(name = "poster_path")
    val posterPath: Any? = null
)